blockchain/for-companies/how-to-run-liberland-company.md

```diff
@@ -0,0 +1,132 @@
+
+[Browse existing Contracts](www.blockchain.liberland.org/home/contracts/overview)
+
+### Choosing personal or company account
+Any person, wallet address of account can create a company in on the Liberland blockchain, however to be officially registered within the nation
+of Liberland it needs to satisfy some minimal requirements.
+
+The first decision is choosing if you want to open a company under your own personal account or create a new company account.
+
+We recommend creating a new company account (wallet address) and doing the process with the company account to keep finances and eventual
+contract obligations separate from you personally. However, depending on the nature of the business you might want to choose to open it under
+your personal account, especially if you are the only physical person in the company.
+
+It is easy to switch between many accounts at [blockchain.liberland.org](blockchain.liberland.org) in the menu wallet selector.
+
+![wallet_selector_list](../media/wallet-selector-list.png)
+
+Whichever account you choose, update your identity, especially display name and contact info in the profile page.
+If you do, this will make it easy for people to send tokens to your address by searching for your display name using the wallet at [blockchain.liberland.org](www.blockchain.liberland.org/home/wallet)
+TODO UPDATE PROFILE LINK
+TODO SEND TOKENS BY DISPLAY NAME PICTURE
+
+
+### Create company
+To create a company, in [blockchain.liberland.org](blockchain.liberland.org) choose registries on the menu and select My companies -> Register company
+
+This will open up a form you can fill out to request company registration.
+Not all fields are mandatory, so feel free to leave some blank if it doesnt apply.
+
+
+Regarding 'Editable by registrar' field, its best to enable it. This means the Liberland company registrar can make changes to your company data
+in case your account is lost, or you are found guilty of breaking the law. Most registrars will not register companies without this field checked.
+
+Sign the transaction and your company request for registration is ready! It should take about a day for the request to be approved.
+
+After it is approved you can get your certificate of incorporation by clicking the button 
+
+TODO GENERATING CERTIFICATE OF INCORPORATION
+
+Your company will also appear in the 
+[list of Liberland companies](www.blockchain.liberland.org/home/registries/allCompanies)
+
+### Create contracts (optional)
+Your company might operate under a legally binding contract, or you might want to sign contracts with clients or other parties.
+You can reference these contracts directly in your company registration if you so wish.
+
+[Browse existing Contracts](www.blockchain.liberland.org/home/contracts/overview)
+
+To create a contract, 
+
+TODO CONTRACT CREATION
+
+Note that contracts might require legal expertise to set it up properly. 
```

Comment:
please replace for: "Please be aware that drafting contracts may necessitate legal expertise specific to the law of Liberland to ensure they are correctly established. Do not hesitate to seek assistance; there may be companies and professionals available who specialize in providing these services."